frontend/src/stores/layout.ts

```diff
@@ -29,13 +29,21 @@ export const useLayoutStore = defineStore("layout", {
     toggleShell() {
       this.showShell = !this.showShell;
     },
+    setCloseOnPrompt(closeFunction: any, onPrompt: string) {
+      let prompt = this.prompts.find((prompt) => prompt.prompt === onPrompt);
+      if (prompt) {
+        prompt.close = closeFunction;
+      }
+      return null;
```

Comment:
the return is unnecessary, it should be a void function that doesnt return anything.